src/utils.js

```diff
@@ -223,6 +223,28 @@ function computeDefaults(
       }, {});
 
     case "array":
+      // inject defaults into existing array defaults
+      if (Array.isArray(defaults)) {
+        defaults = defaults.map((item, idx) => {
+          return computeDefaults(
+            schema.items[idx] || schema.additionalItems || {},
+            item,
+            definitions
+          );
+        });
+      }
+
+      // deeply inject defaults into already existing form data
+      if (Array.isArray(rawFormData)) {
+        defaults = rawFormData.map((array, idx) => {
```

Comment:
Can we rename `array` to `item`? (`array` isn't actually an array, so the name sounds misleading)